Tests/cleanup.py

```diff
@@ -192,6 +222,11 @@ def main(argv):
             prefix = opt[1]
         if opt[0] == "-c" or opt[0] == "--os-cloud":
             cloud = opt[1]
+        if opt[0] == "-i" or opt[0] == "--ipaddr":
+            if "," in opt[1]:
+                ipaddrs = opt[1].split(",")
+            else:
+                ipaddrs = opt[1]
```

Comment:
I don't think this will work. Shouldn't it be `[opt[1]]`? Otherwise, if you iterate, you will get the substrings of length 1...

Reply:
The best solution in my eyes is to always do the split, regardless of whether a comma is contained. The only annoying case is the empty string, which gets us `[""]`. (Matching any IP address, so probably even what is wanted.) However, this case doesn't seem too relevant here.

Reply:
Indeed, adding insult to injury. Fixed with commit https://github.com/SovereignCloudStack/standards/pull/712/commits/28219cf384181f02aa239da8c2737257da013e7b.
Thanks!